packages/api/cms-api/src/dam/files/folders.service.ts

```diff
@@ -153,14 +155,11 @@ export class FoldersService {
         if (parentIsDirty) {
             folder.mpath = folder.parent ? (await this.findAncestorsByParentId(folder.parent.id)).map((f) => f.id) : [];
 
-            await this.foldersRepository
-                .createQueryBuilder()
-                .update({
-                    // TODO is this an attack vector?
-                    mpath: folder.mpath || `mpath[(array_position(mpath, ${folder.id})):array_length(mpath,1)]`,
-                })
-                .where("? = ANY(mpath)", [folder.id])
-                .execute();
+            const connection = this.orm.em.getConnection();
```

Comment:
I'd prefer that we keep using the QueryBuilder and use `raw()`: https://mikro-orm.io/api/next/knex/class/QueryBuilder#raw. Is this possible?